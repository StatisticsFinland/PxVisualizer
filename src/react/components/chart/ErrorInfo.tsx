import React from "react";
import styled from "styled-components";
import { Translations } from "../../../core/conversion/translations";

const ErrorSection = styled.div`
    width: 100%;
    position: relative;
    font-family: 'Barlow Semi Condensed';
`;

export interface IErrorInfoProps {
    locale: string;
}

export const ErrorInfo: React.FC<IErrorInfoProps> = ({ locale }) => {
    return (
        <ErrorSection>
            <h1>{Translations.graphCreationError[locale]}</h1>
        </ErrorSection>
    );
};