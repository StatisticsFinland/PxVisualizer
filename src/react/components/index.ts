export * from "./chart";
export * from "./ErrorBoundary";
export * from "./globalStyle";
export * from "./icon";
export * from "./burgerMenu";
