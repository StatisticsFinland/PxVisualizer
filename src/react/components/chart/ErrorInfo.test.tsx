import React from "react";
import { ErrorInfo } from "./ErrorInfo";
import { render } from "@testing-library/react";

describe('Rendering test', () => {

    it('renders correctly', () => {
        const { asFragment } = render(
            <ErrorInfo locale={'fi'} />
        );
        expect(asFragment()).toMatchSnapshot();
    });
});