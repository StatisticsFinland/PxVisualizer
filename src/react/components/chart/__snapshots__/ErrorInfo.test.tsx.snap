// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering test renders correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-Qotzb ceAfRj"
  >
    <h1>
      Kuviota ei voitu muodostaa
    </h1>
  </div>
</DocumentFragment>
`;
